pkg/volume/csi/csi_mounter.go

```diff
@@ -60,7 +61,7 @@ type csiMountMgr struct {
 	k8s            kubernetes.Interface
 	plugin         *csiPlugin
 	driverName     csiDriverName
-	driverMode     driverMode
+	volumeMode     volumeMode
```

Comment:
If

Reply:
Yes, naming is hard :frowning:

I also considered `ephemeralMode` or `inlineMode`, but both then don't make much sense when the value is `persistent`.

Any suggestions?

Reply:
How about `VolumeSource`, with `persistent` and `ephemeral` as values?

It's then different from `driverMode`, but perhaps that's okay.

Reply:
Yeah, naming things is hard. Slight preference to the previous name, driverMode, as that kind of matches the spec:
CSIDriver.Spec.Mode -> driverMode.

Or maybe csiDriverMode?

Reply:
`VolumeSource` also could be misunderstood (see snapshotting...). Let's see how well `CSIVolumeMode` works...

Reply:
Works for me.

Reply:
I left "driverMode" unchanged, because as you said, it is derived from `CSIDriver.Spec.Mode`.